dp-hello-world-controller/Makefile

```diff
@@ -11,7 +11,11 @@ all: audit test build
 
 .PHONY: audit
 audit:
-	nancy go.sum
+	go list -json -m all | nancy sleuth --exclude-vulnerability-file ./.nancy-ignore
```

Comment:
`-- exclude-vulnerability-file` flag is not needed as the default exclusion file is .nancy-ignore

You would use this flag if you wanted to use a different name to .nancy-ignore